scripts/init-database.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+
+values = list()
+
+
+def retreiveValues():
+    stream = open("../.config/bassa/config.yml", "r")
+    docs = yaml.load_all(stream)
+    global values
+    for doc in docs:
+        for k, v in doc.items():
+            for m, n in v.items():
+                values.append(n)
+
+
+def createDatabase():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]
+    engine = sqlalchemy.create_engine(param)
+    engine.execute("CREATE DATABASE IF NOT EXISTS " + values[3])
+
+
+def importSQL():
+    param = values[5]+'://'+values[0]+':'+values[4]+'@'+values[2]+'/'+values[3]
```

Comment:
Can't we use as `docs.database.database_name`?

Ideally should be like `configs.database.database_name`